src/api/GetAllCountries.js

```diff
@@ -0,0 +1,8 @@
+import RestCountriesApi from './RestCountriesApi.js';
+
+export const GetAllCountries = async () => {
```

Comment:
In Javascript dien je variabelen en functienamen in camelCase te schrijven, evenals de bestandsnamen (behalve componenten, want dat is weer React 😅 )